configuration/pih/htmlforms/hiv/section-hiv-state.xml

```diff
@@ -163,10 +163,10 @@
                     '<lookup expression="encounter.getEncounterDatetime().getTime()"/>',
                     '<uimessage code="pihcore.weeks"/>'
                 );
-            
+
                 setUpNextButtonForSections('hiv-state');
                 setUpDatepickerStartAndEndDateValidation('<uimessage code="pihcore.date.error.inTheFuture"/>','<uimessage code="pihcore.startDate.greater"/>');
-                setUpDatepickerInTheFutureValidation('<uimessage code="pihcore.date.error.inTheFuture"/>');
```

Comment:
I changed this method so that instead of setting up one or more validators of this type, it sets it up only for the one referenced by the passed-in ID, in this case "syph-checkbox"... if we were using this same validation is multiple places on the form, we just call this method twice with the two different IDs.